frontend/packages/pipelines-plugin/src/test-data/pipeline-data.ts

```diff
@@ -1885,6 +1886,115 @@ export const pipelineTestData: PipelineTestData = {
       },
     },
   },
+  [PipelineExampleNames.EMBEDDED_PIPELINE_SPEC]: {
+    dataSource: 'embedded-pipelineSpec',
+    pipeline: {
```

Comment:
```suggestion
    pipeline: null,
```
You're in conflict here. You crafted a Pipeline for a test chunk that says it has an embedded pipeline 🙂

Reply:
it indeed is conflicting. Thanks for the catch.
updated the test data